### PR TITLE
Hit the API

### PR DESCRIPTION
**Pull request by sja-thedude**
- Create a new game with the Leaderboard API using the POST method.
- Implement the "Refresh" button for receiving data from the API.
- Implement the form "Submit" button for sending data to the API.
- Uses async and await JavaScript features to consume the API

### CODE REVIEW

File: src/refresh.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const refresh = () => {
+  const btnRefresh = document.querySelector('.refresh');
+
+  btnRefresh.addEventListener('click', () => {
+    window.location.reload();
+  });
+};
+
+export default refresh;
```
Comment (mikethreels) [inline src/refresh.js]: - [x] good job on the API calls 👍 however according to the requirements When the Refresh button is clicked, it should hit the GET API endpoint causing the scores list to change.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi SJA, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [x] Nice use of arrow functions 💪 however your application is not working when cloning it and running it locally. when I hit the submit button the API doesn't seem to be called and neither is it called when clicking the refresh button.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/refresh.js (hunk 1 above)
Comment (sja-thedude) [reply src/refresh.js]: Thanks, Sir for your feedback! I have made the changes.

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @sja-thedude, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**